libraries/AP_RCProtocol/AP_RCProtocol_FPort.cpp

```diff
@@ -124,8 +124,8 @@ void AP_RCProtocol_FPort::decode_control(const FPort_Frame &frame)
 
     bool failsafe = ((frame.control.flags & (1 << FLAGS_FAILSAFE_BIT)) != 0);
 
-    // we scale rssi by 2x to make it match the value displayed in OpenTX
-    const uint8_t scaled_rssi = MIN(frame.control.rssi*2, 255);
+    // fport rssi 0-50, ardupilot rssi 0-255, scale factor 255/50=5.1
```

Comment:
not a good one for backport

Reply:
This is a nice small non intrusive change.... why not let it in?